### ISSUE TITLE
Bad dependency structure

### ISSUE THREAD

**Issue by nikitinvv** [tomography/tomoscan#148]
Would it be possible to reorganize the tomoscan dependencies structure?

When I do 
`from tomoscan import tomoScanPSO`
in my tomoscan_32id.py then tomoscan imports all subclasses (tomoscan_2bm, tomoscan_7bm). If there is a error in one of these classes or some non-standard dependency then I get this error too..

In the current version, I have the following:
```
>>> from tomoscan import TomoScanPSO
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/beams19/USERTXM/epics/synApps/support/tomoscan/tomoscan/__init__.py", line 16, in <module>
    from tomoscan.tomoscan_7bm import *
  File "/home/beams19/USERTXM/epics/synApps/support/tomoscan/tomoscan/tomoscan_7bm.py", line 17, in <module>
    from tomoscan import TomoScanHelical
ImportError: cannot import name 'TomoScanHelical' from partially initialized module 'tomoscan' (most likely due to a circular import) (/home/beams19/USERTXM/epics/synApps/support/tomoscan/tomoscan/__init__.py)
```
which can be solved by adding 

`from tomoscan.tomoscan_helical import *
`
in __init__.py 

If I would not import tomoscan_7bm.py in __init__.py then I would not get this error..

**Comment by MarkRivers** [comment]
I agree, it would be good to have the final derived classes be independent.

**Comment by decarlof** [comment]
This has been addressed by #149
